cilium/ebpf.io-summit
# Fix/foundation

**Describe what changes this pull request brings**
This pull request adds the changes below for the Foundation page.
- replace slug `/members` with `/foundation`
- update the menu items in Header
- add small styles changes for the section What is eBPF Foundation?  
- adds the section Steering Committee
- adds the section Join as a Member
- update text in the top banner 


**Describe any additional changes**
N/A


**Steps to test**
1. Open [the page](https://deploy-preview-22--ebpf-summit-2021.netlify.app/foundation/)
2. Confirm that everything looks and works as expected


**Screenshots**
Top banner:
![image](https://user-images.githubusercontent.com/48465000/132678264-b442b872-cb39-4eb9-ac1a-1cc36745b851.png)
Header menu:
![image](https://user-images.githubusercontent.com/48465000/132678491-665e73b6-d7bb-4129-b490-0dcb42530a10.png)
Steering Committee:
![image](https://user-images.githubusercontent.com/48465000/132677643-48ac21d6-ce6d-4cc7-87f0-bb89f57af97e.png)
Join as a Member:
![image](https://user-images.githubusercontent.com/48465000/132677728-2170b797-c33b-49d3-9132-33730ddcd6da.png)



**Checklist**



**TODO**


**References**
[Figma](https://www.figma.com/file/DKNPEcutVbDZb1ylyP7wHs/EBPF-Website-Design?node-id=0%3A1)
